src/components/Patient/PatientRegistration.tsx

```diff
@@ -734,10 +734,13 @@ export default function PatientRegistration(
       </div>
       {showDuplicate && (
         <DuplicatePatientDialog
+          open={showDuplicate}
```

Comment:
do we need to do `&&` check when we are already specifying whether to `open` as prop?